test/lib/ex_aws/kms/integration_test.exs

```diff
@@ -0,0 +1,107 @@
+defmodule ExAws.KMSIntegratinTest do
+  use ExUnit.Case, async: true
+
+  defp key_id do
+    System.get_env("AWS_KEY_ARN")
+  end
```

Comment:
`AWS_KEY_ARN` is decided name by me. It is not defacto standard name.  
It might be good to name it for testing just like `TEST_EX_AWS_KEY_ARN`.

The normal app has a `KEY_ARN or KEY_ID` at encrypting by KMS.`KEY_ARN or KEY_ID` is set by the developer.
I guess with the standard approach, environment variables or config files.
So, I use environment variable `AWS_KEY_ARN`.

Do you think about it?